c/clang/_wrap/cursor.cpp

```diff
@@ -1,29 +1,77 @@
-#include <stdio.h>
 #include <clang-c/Index.h>
+#include <stdio.h>
 
-typedef enum CXChildVisitResult(* wrap_CXCursorVisitor) (CXCursor *cursor, CXCursor *parent, CXClientData client_data);
+typedef enum CXChildVisitResult (*wrap_CXCursorVisitor)(CXCursor *cursor, CXCursor *parent, CXClientData client_data);
 
-typedef struct {
-    CXClientData         data;
+typedef struct
+{
+    CXClientData data;
     wrap_CXCursorVisitor visitor;
-} wrap_data;
+} wrapData;
 
-CXChildVisitResult wrap_visitor(CXCursor cursor, CXCursor parent, CXClientData data) {
-    wrap_data *d = (wrap_data*)(data);
-    return d->visitor(&cursor,&parent,d->data);
+CXChildVisitResult wrapVisitor(CXCursor cursor, CXCursor parent, CXClientData data)
+{
+    wrapData *d = (wrapData *)(data);
+    return d->visitor(&cursor, &parent, d->data);
 }
 
-extern "C" {
+extern "C"
+{
 
-CXString wrap_clang_getCursorSpelling(CXCursor *cur) {
-    return clang_getCursorSpelling(*cur);
-}
+    CXString wrapCursorSpelling(CXCursor *cur)
```

Comment:
should have clang namespace. I don't think `wrapCursorSpelling` is better than `wrap_clang_getCursorSpelling`.
bug I think:

```
CXString wrap_clang_getCursorSpelling(CXCursor *cur) {
}
```

is better than:

```
CXString wrap_clang_getCursorSpelling(CXCursor *cur)
{
}
```

Reply:
Alright, I have configured clang-format as follows, which basically aligns with some Go styles:
```
    "C_Cpp.clang_format_style": "{ BasedOnStyle: LLVM, IndentWidth: 4, ColumnLimit: 120, BreakBeforeBraces: Custom, BraceWrapping: { AfterFunction: false } }"
```